src/constructs/lambda/lambda.ts

```diff
@@ -12,7 +12,7 @@ interface ApiProps extends Omit<LambdaRestApiProps, "handler"> {
   id: string;
 }
 
-interface GuFunctionProps extends Omit<FunctionProps, "code"> {
+export interface GuFunctionProps extends Omit<FunctionProps, "code"> {
   code: { bucket: string; key: string };
```

Comment:
Possibly not for this PR, but would be good for the `bucket` to come for free, [GuGetDistributablePolicy](https://github.com/guardian/cdk/blob/dc7c8999c1afaf5bc7b20d89ae4c7421d4c70ec9/src/constructs/iam/policies/s3-get-object.ts#L16-L26) is a vague demonstration of this.